### ISSUE TITLE
Griefer Jail 

### ISSUE THREAD

**Issue by Klaranth** [pandorabox-io/in-game#45]
[SuperGeneral]Envxsion — 08/18/2020
idea: 19k-20k protectionless land which is like regular earth, which can banish excessive pvp/griefers to instead of banning them outright

@白狐 adding on to that, all items are taken from the pvp/griefer and kept with the admin/mods until the banish finishes

**Comment by S-S-X** [comment]
Jail was there and it was not good, jail was removed. I think my game account still has jailer privilege which is leftover from time when there was jail.

Jailing players in game like that does not really help solving problems, chat jailing is bit different as it is normally used with players who cannot behave well in chat or who just do not understand how chat works but are not causing that much other significant troubles.

I believe this might just cause more problems and should not be added again unless there's some strong evidence that it is better way to handle griefers.

**Comment by OgelGames** [comment]
Closing as per SX's comment.

**Comment by Klaranth** [comment]
Marsufoman
prison jail

Marsufoman
Make noobs mine ores to repent for their sins

Huhhila
@MCLV: Somehow you made me wonder about the coordinates of the old jail... if it is still there. Does it have an area number? Anyway if you put jumpdrive:area_engine in crafting grid with jail door, several wrenches and infinite bag of holding, shouldn't it result in something like that? Oh nvm me, vizlib changes deleted area_engine.

MCLV 
I've never even seen the jail. was already decomissioned when I joined

Marsufoman
If there was a jail on here a few years back, i mustve seen it. (i wonder why), however never seen one afaik

Alliecakes 
i think jail is better plugin than ban. But they removed it cos moderator jailed himself. Which i assume showered game with null pointer exceptions

SX 
Well, they kicked random ppl and then banned themself. Jail was removed because it wasn't that good idea in first place. There's discussion about it buried somewhere... but basically it just isn't helpful to keep player in server if you don't want them to be able to do anything, never helpful but potentially can make situation worse.

Alliecakes 
ban appeals though
admin more likely to be on discord despite it being example of non libre software and some people contrarily feel not to pursue usage of Discord, but it managed to muster the numbers

Naso 
About jails: something I've mentioned before that I've seen another server do and that I think is brilliant: jail dimension. A space identical to the surface (but way smaller) but with no rules, no moderation, monthly rollback and respawn anchored to that place. The perfect environment for misbehaving players, where they can misbehave as much as they want with their like minded peers. I don't know how it could be implemented in PB though...
The idea being, instead of making the jail obviously a jail, you move them to what looks like "somewhere in the server" and leave them to figure out what's up. Best case scenario they don't see past the veil and keep living in the matrix

Marsufoman
then they realize

Naso 
And the conversation moves on to discussing how to implement the metaphorical bars of the jail

The4spaceconstants 
That dimension should be the permaban place

Alliecakes 
Minetest engine cannot do dimensions right?

The4spaceconstants 
Transport them to a different server

Alliecakes 
What was worst attack on server u think the hobo gang or maybe make penal colony and revoke interact and regrant on entering and leaving?
